js/views/modals/Settings/SettingsPage.js

```diff
@@ -0,0 +1,72 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import { View } from 'backbone';
+import select2 from 'select2'; // eslint-disable-line no-unused-vars
+
+export default class extends View {
+  constructor(options = {}) {
+    super({
+      className: 'settingsGeneral',
+      events: {
+      },
+      ...options,
+    });
+
+    // temp data. This view will need the user model, the languages, the countries, and the
+    // currencies.
+
+    this.profileModel = {
+      handle: '@example',
+      name: 'exampleName',
+      location: 'UNITED_STATES',
+      about: 'example about',
+      shortDescription: 'example short description',
+      website: 'http://example.website.com',
+      email: 'example@email.com',
+      nsfw: false,
+      vendor: false,
+      moderator: false,
+      primaryColor: '',
+      secondaryColor: '',
+      textColor: '',
+      followerCount: 0,
+      followingCount: 0,
+      listingCount: 0,
+    };
+
+    this.countryList = [
+      { code: 'USA', dataName: 'UNITED_STATES', name: 'United States' },
+      { code: 'DZD', dataName: 'ALGERIA', name: 'Algeria' },
+    ];
+
+    this.options = {
+      // test data
+      countryList: this.countryList,
+    };
+
+    this.options = Object.assign({}, options, this.profileModel, this.options);
```

Comment:
This is a little confusing because normally this.options on a view has to do with "options" related to the view. It seems like you're trying to combine data that you want to provide to the template..?

If that's the case, IMO, then the constructor is not the place to do it. Also, this.options doesn't need to contain data that's intended for the template. I think it's clearer for a view to have it's models separated from it's options, the latter mainly having to do with stuff that's passed in when instantiating the view.

So, if it is indeed a matter of providing the data to the template, this can be done in render:

```
this.$el.html(t({
  countryList: this.countryList,
  ...this.profileModel,
}));
```

(`...<object>` is es7 shorthand to say expand this object here (i.e. merge all of the object's properties here))

(I'm not passing in this.options into the template, because by looking at your genuine options, I don't think you're template needs that info)

and then in the constructor I think all you really need is `this.options = options` if you have a need to have access to any options that were passed in the the view was instantiated.